### PR TITLE
Bug 1418895 Added filtering routes by labels section

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1418895

@ramr @knobunc PTAL

### CODE REVIEW

**Review by ramr — APPROVED** [review]
LGTM

**Comment by bfallonf** [comment]
Great. Thanks @ramr 

@adellape @ahardin-rh @bmcelvee Peer review please?

**Review by knobunc — APPROVED** [review]
LGTM.  Thanks @bfallonf

File: architecture/core_concepts/routes.adoc [hunk 1]
```diff
@@ -234,7 +234,7 @@ $ oc set env dc/router HAPROXY_ROUTER_SYSLOG_ADDRESS=127.0.0.1 HAPROXY_ROUTER_LO
 |`*ROUTER_SYSLOG_ADDRESS*` |  | Address to send log messages. Disabled if empty.
 |`*ROUTER_TCP_BALANCE_SCHEME*` | source | Load-balancing strategy for multiple endpoints for pass-through routes. Available options are `source`, `roundrobin`, or `leastconn`.
 |`*ROUTER_LOAD_BALANCE_ALGORITHM*` | leastconn | Load-balancing strategy routes with multiple endpoints. Available options are `source`, `roundrobin`, and `leastconn`.
-|`*ROUTE_FIELDS*` |  | A field selector to apply to routes to watch, empty means all.
+//|`*ROUTE_FIELDS*` |  | A field selector to apply to routes to watch, empty means all.
```
Comment (ahardin-rh) [inline architecture/core_concepts/routes.adoc]: Is the // intended?

**Comment by ahardin-rh** [comment]
@bfallonf Just one question from me. I'm not sure how the docs look when built locally, so it may be that nothing that needs addressed. :laughing: Everything else LGTM!

File: architecture/core_concepts/routes.adoc (hunk 1 above)
Comment (knobunc) [inline architecture/core_concepts/routes.adoc]: Yes, since @ramr said the ROUTE_FIELDS are not well supported upstream.

**Comment by bfallonf** [comment]
[rev_history]
|xref:../architecture/core_concepts/routes.adoc#architecture-core-concepts-routes[Core Concepts -> Routes]
|Edited out the `ROUTE_LABELS` environment variable to allighn with upstream.
%
|xref:../install_config/router/default_haproxy_router.adoc#install-config-router-default-haproxy[Setting up a Router -> Using the Default HAProxy Router]
|Added the xref:../install_config/router/default_haproxy_router.adoc#install-haproxy-filtering-routes[Filtering Routes to Specific Routers] section.
%

**Comment by bfallonf** [comment]
Thanks @ahardin-rh . I checked, and it builds locally fine. I'll merge :star:

**Pull request merged**